websites/docs-smoke-test/src/content/components/quiz.mdx

```diff
@@ -0,0 +1,9 @@
+---
+title: Quiz component
+---
+
+import { Quiz } from '@commercetools-docs/gatsby-theme-learning'
```

Comment:
do you want to check what's wrong with the shortcodes and why you have to import it here?
or you want to do it in a separate PR ?

Reply:
Will be handled in a separate PR 👍🏼